doc/gui/wxpython/example/README

```diff
@@ -24,12 +24,15 @@ or it can be launched from the console.
 
 3. Edit ./gui/wxpython/Makefile:
 
+-SUBDIRS = docs animation datacatalog mapswipe gmodeler rlisetup psmap dbmgr vdigit rdigit iclass gcp timeline tplot photo2image image2target
++SUBDIRS = docs animation datacatalog example mapswipe gmodeler rlisetup psmap dbmgr vdigit rdigit iclass gcp timeline tplot photo2image image2target
+
  SRCFILES := $(wildcard icons/*.* scripts/* xml/*) \
--	$(wildcard core/* dbmgr/* gcp/* gmodeler/* ... \
-+	$(wildcard core/* dbmgr/* example/* gcp/* gmodeler/* ... \
+-       $(wildcard animation/*.py core/*.py datacatalog/*.py dbmgr/*.py gcp/*.py gmodeler/*.py \
++       $(wildcard animation/*.py core/*.py datacatalog/*.py dbmgr/*.py example/*.py gcp/*.py gmodeler/*.py \
 
--PYDSTDIRS := $(patsubst %,$(ETCDIR)/%,core dbmgr gcp gmodeler ... \
-+PYDSTDIRS := $(patsubst %,$(ETCDIR)/%,core dbmgr example gcp gmodeler ... \
+-PYDSTDIRS := $(patsubst %,$(DSTDIR)/%,animation core datacatalog dbmgr gcp gmodeler \
++PYDSTDIRS := $(patsubst %,$(DSTDIR)/%,animation core datacatalog dbmgr example gcp gmodeler \
```

Comment:
Let's avoid the need to keep this in sync by just saying what needs to be changed. SUBDIRS, SRCFILES, and PYDSTDIRS, or? I guess you need to add that only when you are adding another directory.

Reply:
@wenzeslaus: What do you think then about changing this point to the following?

```
3. Edit ./gui/wxpython/Makefile:

SUBDIRS: Add example
SRCFILES: add example/*py
PYDSTDIRS: add example
```

Reply:
@wenzeslaus Any feedback? Thanks.

Reply:
Yes, that looks like it addressed my comment.

Reply:
@wenzeslaus: Thanks. Addressed in [384d3f3](https://github.com/OSGeo/grass/pull/3233/commits/384d3f39d45f6c8c99d16b6dd8b7bb85797ebd35)

Reply:
Thanks!